Kernel/Arch/riscv64/Firmware/ACPI/StaticParsing.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023, Sönke Holz <sholz8530@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Firmware/ACPI/StaticParsing.h>
+
+namespace Kernel::ACPI::StaticParsing {
+
+ErrorOr<Optional<PhysicalAddress>> find_rsdp_in_platform_specific_memory_locations()
+{
+    // FIXME: Implement finding RSDP for riscv64.
```

Comment:
Is this even a reasonable thing to do on RISC-V? Are vendors using ACPI on RISC-V boards?